src/main/java/frc/robot/RobotContainer.java

```diff
@@ -64,5 +67,9 @@ private void configureButtonBindings() {
             
     }
 
+    public Command getAutonomous() {
+        return new Auto(arm, drive, SmartDashboard.getNumber("Auto Drive Time", 5));
```

Comment:
Can we make this not on smartdashboard (just make it a constant)?

Reply:
B bot should travel 35 feet in 5 seconds at 50% speed, this _probably_ should be a smaller value btw

Reply:
Isn't ChassisSpeeds in m/s? If so then this would be 2.5 meters of movement approximately

Reply:
ohhh, ok that's fine just very slow